src/js/Content/Features/Store/App/CApp.js

```diff
@@ -97,28 +97,39 @@ export class CApp extends CStore {
             FSaveReviewFilters,
         ]);
 
-        this.userNotes = new UserNotes();
-
         this.appid = GameId.getAppid(window.location.host + window.location.pathname);
         this.storeid = `app/${this.appid}`;
 
+        // Some games (e.g. 201270, 201271) have different appid in store page and community
+        this.communityAppid = GameId.getAppidImgSrc(document.querySelector(".apphub_AppIcon img")?.getAttribute("src")) ?? this.appid;
+
+        this.appName = document.querySelector(".apphub_AppName")?.textContent ?? "";
+
+        this.metalink = document.querySelector("#game_area_metalink a")?.getAttribute("href") ?? null;
```

Comment:
Not sure if `?? null` is needed, but I avoided changing the return type just in case.